### DISCUSSION TITLE
[Feature] Expose getByLibraryIdAndOriginalPath to controller

### DISCUSSION THREAD

**Discussion by alvistar** [immich-app/immich#4352] [Feature Request]
### The feature

Hi,
following discussion on https://github.com/immich-app/immich/discussions/4279 to enable the creation of albums from a file structure without resorting to query the db, the following function:

- getByLibraryIdAndOriginalPath from asset.service.ts 

should be exposed in asset.controller.ts.

The patch should be trivial.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by etnoy** [comment]
Completed via #4931
